.github/workflows/build-viz-state.yml

```diff
@@ -0,0 +1,22 @@
+on:
+  workflow_call:
+
+jobs:
+  viz-state:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: latest-result
+      - uses: Swatinem/rust-cache@v2
+      - uses: cachix/install-nix-action@v19
+        with:
+          github_access_token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v3
+        with:
+          repository: runziggurat/crunchy
+      - run: cargo run --release -- -i latest.json -o latest.viz.json
+      - uses: actions/upload-artifact@v3
+        with:
+          name: latest-result
+          path: latest.viz.json
```

Comment:
Some second thoughts about this: maybe we should actually call another `process-results` here with `name: crunchy` or `name: p2p-viz`. Somewhat related to this: https://github.com/runziggurat/zcash/pull/234#discussion_r1198834654

Reply:
Wrote a comment there as well https://github.com/runziggurat/zcash/pull/234#discussion_r1198841477.

Currently, we do not save the copy of gzipped `latest.viz.json`. `process-results` saves the gzipped copy of `latest.$EXTENSION` only, and uploads other files as is. This works as we add both files to the same artifact since they go to the same place in GCS, and all of it works in on step.

Reply:
Gotcha! I think it's fine to store the state files under `crawler/` as long as we do not capture historical data.